syncd/tests.cpp

```diff
@@ -38,21 +38,6 @@ extern "C" {
 using namespace syncd;
 
 
-// TODO remove when SAI will introduce bulk APIs to those objects
-sai_status_t sai_bulk_create_fdb_entry(
```

Comment:
Do you want these changes again ?.

Reply:
No. This is not part of voq implementation. I'll remove this change from this PR.